schema/interface.ipv4.yml

```diff
@@ -106,3 +120,60 @@ properties:
     type: array
     items:
       $ref: "https://ucentral.io/schema/v1/interface/ipv4/port-forward/"
+  multicast:
+    type: object
+    properties:
+      unknown-multicast-flood-control:
+        description:
+          The unknown multicast flood control feature enables the system to forward unknown multicast packets only to a multicast router (mrouter).
+        type: boolean
+        default: true
+      igmp:
+        type: object
+        properties:
+          snooping-enable:
+            description:
+              Enable or disable IGMP snooping on per-VLAN basis.
+            type: boolean
+            default: true
+          version:
+            description:
+              Configures the IGMP version. Configurable versions are: IGMPv1, IGMPv2, and IGMPv3
```

Comment:
The ':' in the description comments causes the description in ucentral.schema.full.json to be an object, which messes up the schema, remove the ':' that precedes IGMPv1 and you should be good